homeassistant/components/sensor/sense.py

```diff
@@ -0,0 +1,114 @@
+"""
+Support for monitoring a Sense energy sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sense/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_EMAIL, CONF_PASSWORD)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['sense_energy==0.2.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+ACTIVE_NAME = "Energy Usage"
+ACTIVE_SOLAR_NAME = "Solar Energy Generation"
+DAILY_NAME = "Daily Energy Usage"
+
+ACTIVE_TYPE = 'active'
+ACTIVE_SOLAR_TYPE = 'solar'
+DAILY_TYPE = 'daily'
+
+ICON = 'mdi:flash'
+
+MIN_TIME_BETWEEN_DAILY_UPDATES = timedelta(seconds=150)
+MIN_TIME_BETWEEN_ACTIVE_UPDATES = timedelta(seconds=10)
```

Comment:
This is pretty fast for a remote API. With those intervals the sensor will create around 280000 requests per month and instance.